src/Polytopes/Cone/properties.jl

```diff
@@ -217,11 +176,11 @@ This gives us a 1-dimensional lineality.
 julia> UH = Cone([1 0; 0 1; -1 0]);
 
 julia> lineality_space(UH)
-pm::Matrix<pm::Rational>
-1 0
+1-element VectorIterator{RayVector{Polymake.Rational}}:
+ [1, 0]
 ```
 """
-lineality_space(C::Cone) = pm_cone(C).LINEALITY_SPACE
+lineality_space(C::Cone) = VectorIterator{RayVector{Polymake.Rational}}(pm_cone(C).LINEALITY_SPACE)
```

Comment:
So one question I have is whether we should introduce a different element type for the lineality space, since mathematically these are not `RayVector`s, but more like `LinealityGenerator` or `LineVector`. For now this is fine though.

Reply:
I agree. I was stumbling upon this on implementation but decided to keep things simple at first. Adding this now is possible anytime.